later/lib/database/datasources/records_local_data_source.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:injectable/injectable.dart';
+import 'package:isar/isar.dart';
+import 'package:later/database/models/record_model.dart';
+
+import './records_data_source.dart';
+
+@LazySingleton(as: RecordsDataSource)
+class RecordsLocalDataSource extends RecordsDataSource {
+  final Isar _isar;
+
+  const RecordsLocalDataSource(this._isar);
+
+  @override
+  Future<RecordDbModel> create({
+    required String url,
+    required String title,
+    String? description,
+    DateTime? lastEditedAt,
```

Comment:
I don't think that `lastEditedAt` must be settable like this... We should only set it when we edit a record in methods like `update` etc.